client/src/Controller/Admin/Client/SearchController.php

```diff
@@ -83,4 +104,18 @@ private function getDefaultFilters(Request $request): array
             'sort_order' => 'DESC',
         ];
     }
+
+    private function formatAndSortResults(array $results)
+    {
+        //Filter out non-unique values (case insensitive)
+        $serialized = array_map('serialize', $results);
+        $unique = array_intersect_key($serialized, array_unique(array_map('strtolower', $serialized)));
+        $filteredResults = array_intersect_key($results, $unique);
+
+        //Sort alphabetically (case insensitive)
+        $column = array_column($filteredResults, 'clientSurname');
+        array_multisort($column, SORT_ASC, SORT_NATURAL | SORT_FLAG_CASE, $filteredResults);
+
+        return $filteredResults;
```

Comment:
Hmmmm - I'd be interested to see how this performs when we have a very large dataset. It could be fine but any sort of ordering and filtering I tend to offload to SQL as it's so much quicker than doing it in code.